src/start.cli.ts

```diff
@@ -85,6 +86,36 @@ yargs.completion("completion")
         }
 
     })
+    .command(["gql-introspect <team>"], "Introspect GraphQL schema", ya => {
```

Comment:
While "introspect" is what apollo uses, it seems a bit abstract to me. Would "fetch" or something like that be more clear?